core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/sync/EccNodesSync.java

```diff
@@ -0,0 +1,226 @@
+        myStatementDecorator = Preconditions
+                .checkNotNull(builder.myStatementDecorator, "StatementDecorator cannot be null");
+        myNodesList = Preconditions
+            .checkNotNull(builder.initialNodesList, "Nodes list cannot be null");
+        myCreateStatement = mySession.prepare(QueryBuilder.insertInto(KEYSPACE_NAME, TABLE_NAME)
+                .value(COLUMN_ECCHRONOS_ID, bindMarker())
+                .value(COLUMN_DC_NAME, bindMarker())
+                .value(COLUMN_NODE_ENDPOINT, bindMarker())
+                .value(COLUMN_NODE_STATUS, bindMarker())
+                .value(COLUMN_LAST_CONNECTION, bindMarker())
+                .value(COLUMN_NEXT_CONNECTION, bindMarker())
+                .value(COLUMN_NODE_ID, bindMarker())
+                .build()
+                .setConsistencyLevel(ConsistencyLevel.LOCAL_QUORUM));
+        ecChronosID = ECCHORONS_ID_PRE_STRING.concat(InetAddress.getLocalHost().getHostName());
+    }
+
+    public List<ResultSet> acquireNodes() throws EcChronosException
+    {
+        List<ResultSet> resultSets = new ArrayList<>();
+        if (!myNodesList.isEmpty())
+        {
+            for (Node node : myNodesList)
+            {
+                LOG.info(
+                    "Preparing to acquire node {} with endpoint {} and Datacenter {}",
+                    node.getHostId(),
+                    node.getEndPoint(),
+                    node.getDatacenter());
+                ResultSet tmpResultSet = acquireNode(node);
+                LOG.info("Node successfully acquired by instance {}", ecChronosID);
+                resultSets.add(tmpResultSet);
+            }
+        }
+        else
+        {
+            throw new EcChronosException("Cannot Acquire Nodes because there is no nodes to be acquired");
+        }
+        return resultSets;
+    }
+
+    private ResultSet acquireNode(final Node node)
+    {
+        return insertNodeInfo(
+            ecChronosID,
+            node.getDatacenter(),
+            node.getEndPoint().toString(),
+            node.getState().toString(),
+            Instant.now(),
+            Instant.now().plus(DEFAULT_CONNECTION_DELAY_IN_MINUTES, ChronoUnit.MINUTES),
+            node.getHostId());
+    }
+
+    @VisibleForTesting
+    public ResultSet verifyAquireNode(final Node node)
+    {
+        return acquireNode(node);
+    }
+
+    private ResultSet insertNodeInfo(
+        final String ecchronosID,
+        final String datacenterName,
+        final String nodeEndpoint,
+        final String nodeStatus,
+        final Instant lastConnection,
+        final Instant nextConnection,
+        final UUID nodeID)
+    {
+        BoundStatement insertNodeSyncInfo = myCreateStatement.bind(ecchronosID,
+                datacenterName, nodeEndpoint, nodeStatus, lastConnection, nextConnection, nodeID);
+        return execute(insertNodeSyncInfo);
+    }
+
```

Comment:
Not sure what the point is for this method, as it just passes all attributes through to another method.